src/plugins/intel_gpu/include/intel_gpu/primitives/roi_align.hpp

```diff
@@ -19,10 +19,10 @@ struct roi_align : public primitive_base<roi_align> {
     CLDNN_DECLARE_PRIMITIVE(roi_align)
 
     /// @brief Pooling mode for the @ref roi_align
-    enum PoolingMode {
-        Max,
-        Avg
-    };
+    enum PoolingMode { Max, Avg };
+
+    /// @brief Aligned mode for the @ref roi_align
+    enum AlignedMode { Asymmetric, Half_pixel_for_nn, Half_pixel };
```

Comment:
nit: lower case first letter is used in most of the other enums, so it's better to use similar coding style

Reply:
Done